README.md

```diff
@@ -698,6 +698,13 @@ generic methods.
     See _sip_ scheme.  Its syntax is the same as sip, but the default
     port is different.
 
+- **smb**:
+
+    `URI` objects belonging to the smb scheme support the common,
+    generic and server methods. In addition, they provide methods to
+    access the userinfo sub-components ($uri->user and $uri->password)
+    as well as $uri->authdomain and $uri->sharename methods.
+
```

Comment:
This change needs to go into lib/URI.pm please, the README.md is generated automatically by dzil.